optimizely/helpers/event_tag_utils.py

```diff
@@ -116,15 +114,14 @@ def get_numeric_value(event_tags, logger=None):
         if logger:
             logger.log(
                 enums.LogLevels.INFO,
-                'The numeric metric value {} will be sent to results.'.format(numeric_metric_value),
+                f'The numeric metric value {numeric_metric_value} will be sent to results.'
             )
     else:
         if logger:
             logger.log(
                 enums.LogLevels.WARNING,
-                'The provided numeric metric value {} is in an invalid format and will not be sent to results.'.format(
-                    numeric_metric_value
-                ),
+                f'The provided numeric metric value {numeric_metric_value}'
+                ' is in an invalid format and will not be sent to results.'
```

Comment:
is this all in one line? If two lines you will need to break with a backslash and add f in the beginning of the second line